pkg/controller/node/node_controller.go

```diff
@@ -90,6 +101,10 @@ func (n *NodeController) SyncHandler(key string) error {
 	if _, ok := node.Annotations[nodeEgressIPConfigAnnotationKey]; ok {
 		return nil
 	}
+	// Skip synchronization if this node is still uninitialized by the Cloud Controller Manager.
+	if !isNodeInitializedByCloudControllerManager(node) {
```

Comment:
instead of using the helper method can we just add:
```
// check if the node was already initialized by the Cloud Controller Manager
// https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/
if _, ok := if !node.Spec.Taints[cloudproviderapi.TaintExternalCloudProvider] {
 // LOG
return nil
}
```